src/main/java/baritone/pathing/movement/movements/MovementParkour.java

```diff
@@ -110,7 +111,7 @@ public static void cost(CalculationContext context, int x, int y, int z, EnumFac
                     return;
                 }
             }
-            if (MovementHelper.canWalkOn(context, x + xDiff * i, y - 1, z + zDiff * i)) {
+            if (MovementHelper.canWalkOn(context.bsi(), x + xDiff * i, y - 1, z + zDiff * i)) {
```

Comment:
This is just icky... why?

Reply:
Part of cost calculation, wanted to reduce amount of calls while keeping canWalkOn usable by updateState, which doesn't have a calc context

Reply:
Fair